src/import_resolver.rs

```diff
@@ -18,6 +20,16 @@ use crate::path_utils::{as_slashed_pathbuf, slashed_as_relative_path};
 pub struct TsconfigPathsJson {
     pub compiler_options: TsconfigPathsCompilerOptions,
 }
+
+impl TsconfigPathsJson {
+    pub fn from_path(tsconfig_path: String) -> Self {
+        let file = File::open(tsconfig_path).unwrap();
+        let buf_reader = BufReader::new(file);
+        let tsconfig_paths_json: TsconfigPathsJson = serde_json::from_reader(buf_reader).unwrap();
```

Comment:
This should return Result<Tsconfig, Err> rather than unwrapping. It's safe to unwrap at all the current callsites (tests and in main())